.appveyor.yml

```diff
@@ -0,0 +1,38 @@
+version: '{build}'
+
+# Build Using Our Hardware And Image
+build_cloud: "unwitting-accomplice Docker"
```

Comment:
non-blocking comment, but _maybe_ would be better if our machine was named more descriptively just so it's super clear what this is referring to for future people.

Reply:
At least added a comment about where the server is.